docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -83,10 +139,10 @@ Connecting to ElasticSearch ReST API (Flows)   [ Success  ] => Not configured
 => Everything is awesome
 ----
 
-Check Karaf logs if the registration fails or the health check does not say `SERVING`.
+Check the Karaf logs if the registration fails or the health check does not say `SERVING`.
 If this is the case, you will need to open a support ticket.
 
-You can also check that `OPENNMS_HOME/share/rrd` files are still being updated.
+You can also check if the `$\{OPENNMS_HOME}/share/rrd` files are still being updated.
 If they are not, this indicates that the Time Series DB is not working.
```

Comment:
@drewym @jeffgdotorg Can either of you confirm this? It seems like the opposite should be true.  If there are NO new updates to the RRD files, then the integration plugin would be handling metric storage.